application/comit_node/src/swap_protocols/rfc003/state_machine.rs

```diff
@@ -1,4 +1,5 @@
-#![allow(clippy::too_many_arguments)] // TODO: Figure out how to properly place this on the state_machine_future derive so that is is forwarded to the generated structs and impl
+#![allow(clippy::too_many_arguments)] // TODO: Figure out how to properly place this on the state_machine_future
```

Comment:
:upside_down_face: Can we move this ontop of the `allow` as well same as you did with redundant closure one?

Maybe check if it is even still valid since we changed the SM quite a bit.

Reply:
Unfortunately
```
warning: this function has too many arguments (10/7)
   --> application/comit_node/src/swap_protocols/rfc003/state_machine.rs:178:27
    |
178 | #[derive(StateMachineFuture)]
    |                           ^
    |
```
but I put the `TODO` on top.